akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -322,6 +340,7 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
     case None         ⇒ remoteSettings.AeronDirectoryName
   }
 
+  // TODO: Add FR events
```

Comment:
Yes, we will add instrumentation as we need them. I don't want to do it in this PR.